docs/examples/components/details.yaml

```diff
@@ -0,0 +1,33 @@
+componentName: details
+description: The details component handles the flow for device details.
+
+emitters:
+  - message: kafmesh.deviceId.detail
```

Comment:
this is both protobuf schema info and kafka topic right?

Reply:
yeah. I'm thinking of making a topic name override in case you wan to break out of the mold